roadmap.md

```diff
@@ -42,7 +42,7 @@ item; they don't imply an order for implementation._
     1. Events are sent by code written by Developer1 and received by code
        written by Developer2, where Developer1 has no knowledge of Developer2.
 
-*0.2*
+*0.2* - Completed - 2018/12/05
```

Comment:
```suggestion
*0.2* - Completed - 2018/12/06
```

Super-minor, but I think the vote officially closes tomorrow? 😉

Reply:
clearly calendaring isn't my thing :-)  will fix, thanks